server.js

```diff
@@ -762,7 +791,18 @@ app.get('/questionpreview', function(req, res) {
             user: 'Username',
             question: q,
             answered: false,
-            preview: true
+            preview: true,
+            getQuestionForm: function(){
```

Comment:
This section seems to be repeated a few times, is there anything we can do about that?

Reply:
its not repeated they are all different depending on what view you want to see

Reply:
There is still duplication in the return values, this should be unified in a function